### PR TITLE
Updated Code Compliant Dataset to Reflect Approved Changes in ASHRAE 90.1

### PR DESCRIPTION
**Pull request by aowabinr**
Pull request overview
---------------------
Update the code compliant equipment dataset to reflect approved changes to ASHRAE Standard 90.1.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [X] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [X] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish

### Reviewer
This will not be exhaustively relevant to every PR.
 - [x] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [x] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by lymereJ** [comment]
The previous coefficients were part of a change code proposal to ASHRAE 90.1. The new and proposed coefficients have been reviewed and approved by the ASHRAE 90.1 Mechanical Subcommittee and made their way into [addendum bd to 90.1-2019](https://www.ashrae.org/file%20library/technical%20resources/standards%20and%20guidelines/standards%20addenda/90_1_2019_bd_20220531.pdf). The addendum introduces a new normative Appendix (J) which includes the coefficients. The new appendix will be included in 90.1-2022.

File: cmake/Install.cmake [hunk 1]
```diff
@@ -268,6 +268,7 @@ install(FILES "${PROJECT_SOURCE_DIR}/datasets/ASHRAE_2005_HOF_Materials.idf" DES
 install(FILES "${PROJECT_SOURCE_DIR}/datasets/Boilers.idf" DESTINATION "./DataSets" COMPONENT Datasets)
 install(FILES "${PROJECT_SOURCE_DIR}/datasets/California_Title_24-2008.idf" DESTINATION "./DataSets" COMPONENT Datasets)
 install(FILES "${PROJECT_SOURCE_DIR}/datasets/Chillers.idf" DESTINATION "./DataSets" COMPONENT Datasets)
+install(FILES "${PROJECT_SOURCE_DIR}/datasets/CodeCompliantEquipment.idf" DESTINATION "./DataSets" COMPONENT Datasets)
```
Comment (lymereJ) [inline cmake/Install.cmake:271]: #9046 added the dataset but failed to add it here to be installed.

**Comment by rraustad** [comment]
@aowabinr I am not up to speed on these new 90.1 performance curves. Can you give a summery of why/how these curves improve simulation accuracy?

**Comment by aowabinr** [comment]
@rraustad  The updated performance curves are not intended to improve the accuracy of simulations specifically. We generated the curves to regulate the performance of chillers in the baseline models generated through the Appendix G rules, or ECB (Chapter 11). We developed the curves using a methodology such that the curves meet both the full and part load efficiency requirements. These curves have been approved and are going to be published in 90.1-2022 in Appendix J.

**Comment by rraustad** [comment]
Well, then I would think we would want to keep the older 2019 curves and add the 2022 curves? @mjwitte ?

**Comment by lymereJ** [comment]
In reality there is no 2019 curves. The initial sets of curves were created and submitted as an addendum to 90.1-2019. After getting feedback from the ASHRAE 90.1 Mechanical Sub-Committee (MSC), they were revised. The curves included in this PR are the revised ones that have been approved by the MSC and that are going to be included in Appendix J to 90.1-2022.

I'm not opposed to keep the older curves but because they are not referenced by the standard and are really similar to the ones we're proposing here, I thought that it might be confusing for users. What do you think?

**Comment by rraustad** [comment]
I see. OK, that makes sense. This can merge since the CodeCompliantEquipment.idf doc is specific to the ASHRAE 90.1 2022 complicance curves and previous curves were not referenced by any standard.

**Comment by rraustad** [comment]
Review shows curves are available for:

    Chiller:Electric:EIR,
      A2 , \field Cooling Capacity Function of Temperature Curve Name
      A3 , \field Electric Input to Cooling Output Ratio Function of Temperature Curve Name
      A4 , \field Electric Input to Cooling Output Ratio Function of Part Load Ratio Curve Name

**Review by rraustad — APPROVED** [review]
Accepted as only available compilance curves.

**Comment by Myoldmopar** [comment]
@aowabinr have you completed a contribution agreement?  The contribution agreement can be found [here](https://energyplus.net/contributing), and before we can accept these changes, they need to be granted to us under a contribution agreement.  If you have questions, please email me at edwin dot lee at nrel dot gov.  Thanks!

**Comment by Myoldmopar** [comment]
I got the word that this is coming through your work at PNNL, so no contribution agreement is needed.  I'm gonna go ahead and merge this in, thanks.

**Pull request merged**